frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ProjectActionsForm.tsx

```diff
@@ -138,6 +219,98 @@ export const ProjectActionsForm = ({
                 onBlur={(e) => handleOnBlur(() => validateName(e.target.value))}
                 autoComplete='off'
             />
+
+            <Step name='Trigger'>
+                <StyledInputDescription>
+                    Create incoming webhooks from&nbsp;
+                    <RouterLink to='/integrations/incoming-webhooks'>
+                        integrations section
+                    </RouterLink>
+                    .
+                </StyledInputDescription>
+                <GeneralSelect
+                    label='Incoming webhook'
+                    name='incoming-webhook'
+                    options={incomingWebhookOptions}
+                    value={`${sourceId}`}
+                    onChange={(v) => {
+                        setSourceId(parseInt(v));
+                    }}
+                />
+            </Step>
+
+            <Step name='When this'>
+                {filters.map((filter, index) => (
+                    <Filter
+                        index={index}
+                        filter={filter}
+                        stateChanged={updateInFilters}
+                        onDelete={() =>
+                            setFilters((filters) =>
+                                filters.filter((f) => f.id !== filter.id),
+                            )
+                        }
+                    />
+                ))}
+
+                <hr />
```

Comment:
These look a bit weird on the UI. Should we remove them?

Reply:
If you help me to add additional space in between the components